cristin-organization/build.gradle

```diff
@@ -28,10 +37,14 @@ task karateTest(type: Test) {
 
     environment "ALLOWED_ORIGIN", "*"
     environment "CRISTIN_API_HOST", "api.cristin.no"
-    environment "BASE_PATH", "cristin"
+    environment "BASE_PATH", "cristin-karate-exclude"
```

Comment:
Why this?

Reply:
Could be anything, just to make it point any other place than the 'main' mapping